eth1/goeth/goETH.go

```diff
@@ -118,11 +118,13 @@ func (ec *eth1Client) HealthCheck() []string {
 	}
 	if sp != nil {
 		reportNodeStatus(statusSyncing)
+		metricsEth1LastSyncedBlock.Set(float64(sp.CurrentBlock))
```

Comment:
this current block is the latest block that the ETH node is familiar with, we need to use the latest block that contains a registry event (e.g. register validator)